apps/site/src/views/Resources/components/HeadingCard/HeadingCard.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "zothacks-theme" as theme;
+
+$item-padding-y: 32px;
+$item-padding-x: 48px;
```

Comment:
Thought: similar to the FAQs, on smaller screen sizes the text looks a bit squished.